spring-cloud-dataflow-docs/pom.xml

```diff
@@ -92,21 +101,11 @@
 												<include name="**/*.html" />
 											</fileset>
 										</copy>
-										<copy todir="${basedir}/target/contents/reference/htmlsingle">
-											<fileset dir="${basedir}/src/main/docbook">
-												<include name="**/*.css" />
-												<include name="**/*.png" />
-												<include name="**/*.gif" />
-												<include name="**/*.jpg" />
-											</fileset>
+										<copy todir="${basedir}/target/contents/reference/htmlsingle/images">
+											<fileset dir="${basedir}/target/generated-docs/images"/>
```

Comment:
This seems to render this step dependent on the docbook one. Couldn't we have images originally in a non-flavored directory (ie *not* `src/main/docbook/images`) and use that everywhere?